### PR TITLE
[DI2-188] Add a new section to the feedback modal to request new features

### PR DESCRIPTION
**Pull request by jonkoops**

### CODE REVIEW

File: src/app/components/Modal/FeedbackModal.jsx [hunk 1]
```diff
@@ -83,23 +83,37 @@ const FeedbackModalComponent = ({
     </ModalBlock>
     <Divider gutter />
     <ModalBlock>
-      <Heading as="h4">Probleem of suggestie?</Heading>
+      <Heading as="h4">Verzoek nieuwe data of functionaliteiten</Heading>
+      <Paragraph>
+        Heb je een verzoek voor bijvoorbeeld een nieuwe dataset, kaartlaag of een verbetering van de
+        site? We horen het graag.
+      </Paragraph>
+      <Button
```
Comment (alimpens) [inline src/app/components/Modal/FeedbackModal.jsx:91]: Does it make sense that you leave this page when you click this button? The other buttons open your mail program (if you have one), you don't leave the page.

Comment (jonkoops) [reply src/app/components/Modal/FeedbackModal.jsx:91]: Yeah good point, should I add a blank target here so it opens in a new tab?

Comment (alimpens) [reply src/app/components/Modal/FeedbackModal.jsx:91]: Makes sense to me, but maybe check with UX

**Review by chrisvanmook — APPROVED** [review]

File: src/app/components/Modal/FeedbackModal.jsx (hunk 1 above)
Comment (jonkoops) [reply src/app/components/Modal/FeedbackModal.jsx:91]: Checked with Martijn and this is accepted behaviour

**Pull request merged**